wagtailvideos/views/videos.py

```diff
@@ -98,14 +96,10 @@ def edit(request, video_id):
         ).format(video.title), buttons=[
             messages.button(reverse('wagtailvideos:delete', args=(video.id,)), _('Delete'))
         ])
-    if is_modeladmin_installed():
-        url_helper = AdminURLHelper(Video.get_track_listing_model())
-        if hasattr(video, 'track_listing'):
-            action_url = url_helper.get_action_url('edit', instance_pk=video.track_listing.pk)
-        else:
-            action_url = url_helper.create_url
+    if hasattr(video, 'track_listing'):
+        action_url = reverse('wagtailvideos_tracks:edit', args=(video.track_listing.pk,))
     else:
-        action_url = ''
+        action_url = reverse('wagtailvideos_tracks:create', args=(video.pk,))
```

Comment:
`wagtailvideos_tracks:create` doesn't exist, so this breaks the video summary when viewed without an attached track listing. `wagtailvideos_tracks:add` exists and doesn't take any arguments